db/api/v1/controllers/projects.controller.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Maps Project related requests to the correct database function and returns a response.
+ * 
+ * TODO: License?
+ */
+
+const projectsDb = require('../db/projects.db');
+
+/**
+ * Finds a Project in the database.
+ * 
+ * @param {*} req - The incoming Express request
+ * @param {*} res - The outgoing Express Response
+ * @param {*} next - The next Express middleware function in the stack
+ */
+const find = async (req, res, next) => {
+    // const sponsor_name = req.valid.sponsor_name;
```

Comment:
Should this be removed?

Reply:
Will remove it and we can consider adding a search by sponsor name feature to the backlog